liferay-gradle/blade.reluctant.service.reference.override/blade.reluctant.service.reference.custom/README.markdown

```diff
@@ -0,0 +1,22 @@
+# Overriding Service Reference Module
```

Comment:
So far we have kept all of our samples as top-level directories (i.e. no nesting) could you update this one where the two projects are children of `liferay-gradle` project?